cmd/sensor-config/main.go

```diff
@@ -125,12 +127,26 @@ func main() {
 		}
 		defer file.Close()
 
-		if err := network.MarshalIndent(file, "", "  "); err != nil {
-			log.Fatalf("unable to marshal output file %q: %v", settings.output, err)
+		switch {
+		case settings.json:
+			if err := network.EncodeJSON(file); err != nil {
+				log.Fatalf("unable to marshal output: %v", err)
```

Comment:
question: why not same log as the `default` case?

Reply:
just being copy/paste missing bits